contents/scalability/scalable-development.md

```diff
@@ -0,0 +1,71 @@
+# Scalable Development - An Introduction
+
+Authors: Loh Jia Shun Kenneth
+
+## Scalable Development
+When Pokémon Go first launched in July 6, the heavy traffic from users caused its servers to crash. However, as of 2017, Google handles at least 3.5 billion searches daily, Whatsapp handles at least 10 billion messages a day, and Facebook hosts 1.2 billion daily active users. What makes these companies different?
+
+The key difference is their server infrastructure. A scalable server allows a company to provide reliable service even as the user base increases. Even if your service might not have to cater to millions of users at the moment, designing scalable software now will reduce the need to restructure the entire backend when the need arises.
+
+## Scalability
+
+So what is scalability? Scalability is the capacity to which a system can cope with an increased workload. Workload can refer to number of users on the server, number of files uploaded, number of concurrent requests, or any quantity that the system needs to scale for.
```

Comment:
The preceding paragraph uses the term `scalability` a few time already before you define it here.